<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="mainLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="controllLayout">
        <item>
         <widget class="QFrame" name="inputFrame">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>410</width>
            <height>250</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>391</width>
             <height>231</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="inputLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QListWidget" name="inputFilesListWidget">
              <property name="toolTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="aptButtonLayout">
          <item>
           <widget class="QPushButton" name="aptImportButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="aptClearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="aptExportButton">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="changeValidStylesPushButton">
          <property name="text">
           <string>Change Valid Styles (affecting Import)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="homeAptLayout">
          <item row="3" column="1">
           <widget class="QSlider" name="infoAptHorizontalSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>1500</number>
            </property>
            <property name="value">
             <number>450</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="selectAptPushButton">
            <property name="text">
             <string>Select Airport</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="amountRadioButton">
            <property name="text">
             <string>Amount of Airports</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="aptLabel">
            <property name="text">
             <string> - </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="distanceRadioButton">
            <property name="text">
             <string>Distance in km</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="infoAptLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Distance in km:  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="InfoAptSpinBox">
              <property name="maximum">
               <number>1500</number>
              </property>
              <property name="value">
               <number>450</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="applyAptPushButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="aptTableWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>code</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>country</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>lat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>lon</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>elev</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>style</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>rwdir</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>rwlen</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>freq</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>desc</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionAddInputFile"/>
   </widget>
   <widget class="QMenu" name="menuAirports">
    <property name="title">
     <string>Airports</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionClear"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAirports"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddInputFile">
   <property name="text">
    <string>Add Input File</string>
   </property>
  </action>
  <action name="actionSave_Output_File">
   <property name="text">
    <string>Save Output File</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>InfoAptSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>infoAptHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoAptHorizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>InfoAptSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
